requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/metabrainz/brainzutils-python.git@v1.10.1
+git+git://github.com/spellew/brainzutils-python.git@additional-entity-support#egg=brainzutils
```

Comment:
@spellew There was a release made after we merged your pull request. You can use release version: `v1.12.0` for BU for these changes.